src/components/inline-alert/index.tsx

```diff
@@ -0,0 +1,25 @@
+import classNames from 'classnames'
+import { IconInfo24 } from '@hashicorp/flight-icons/svg-react/info-24'
+import { InlineAlertProps } from './types'
+import s from './inline-alert.module.css'
+
+export default function InlineAlert({
+	color = 'neutral',
+	description,
+	icon,
+	title,
+}: InlineAlertProps) {
+	return (
+		<div className={classNames(s.default, s[color])}>
+			<span className={s.icon} data-testid="icon">
+				{icon ?? <IconInfo24 />}
+			</span>
+			<span className={s.content}>
+				<p className={s.title}>{title}</p>
+				<span className={s.description}>{description}</span>
```

Comment:
Still on the fence here whether I prefer this to be a prop or just `children`. I'm fine with this for now but I feel strongly the mdx component should accept `children` for ease of authoring.